sympy/physics/mechanics/_system.py

```diff
@@ -798,20 +798,42 @@ def form_eoms(self, eom_method=KanesMethod):
         # KanesMethod does not accept empty iterables
         loads = self.loads if self.loads else None
         if issubclass(eom_method, KanesMethod):
+            not_allowed_kwargs = {
+                "frame", "q_ind", "u_ind", "kd_eqs", "q_dependent",
+                "u_dependent", "configuration_constraints",
+                "velocity_constraints", "forcelist", "bodies"}
+            wrong_kwargs = not_allowed_kwargs.intersection(kwargs)
+            if wrong_kwargs:
+                raise ValueError(
+                    f"The key word arguments {wrong_kwargs} are not allowed to "
```

Comment:
```suggestion
                    f"The keyword arguments {wrong_kwargs} are not allowed to "
```

Reply:
Addressed in 1c4af023695dcaf60f2edfb87350183363e84b42